src/PropertiedObject.h

```diff
@@ -11,6 +11,7 @@ class LuaManager;
 class PropertiedObject {
 public:
 	PropertyMap &Properties() { return m_properties; }
+	const PropertyMap &Properties() const { return m_properties; }
```

Comment:
Why do you want ot pass it as const? I actually kind of like it flagged as mutable, since even though the C++ structure won't be changed, the reference Lua table can.